api/base/views.py

```diff
@@ -455,6 +456,33 @@ def error_404(request, format=None, *args, **kwargs):
     )
 
 
+class BaseChildrenList(JSONAPIBaseView):
+    permission_classes = (
+        ContributorOrPublic,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        ReadOnlyIfRegistration,
+        base_permissions.TokenHasScope,
+        ExcludeWithdrawals,
+    )
+    ordering = ('-modified',)
+
+    def get_default_queryset(self):
```

Comment:
Is it worth noting this is from `ListFilterMixin`? I had to search for it to figure it out, since it's not a DRF thing and `ListFilterMixin` isn't directly used by this class.

Reply:
...on that note, might make sense to do
```py
class BaseChildrenList(JSONAPIBaseView, NodesFilterMixin):
```
since this assumes the filter mixin is there.

Reply:
I don't know if order is going to matter, but i'll run tests and see.